docs/cookbook/sortAbsoluteInstants.mjs

```diff
@@ -0,0 +1,34 @@
+import assert from "assert";
+
+/**
+ * getSortedInstants will sort an array of strings (each of which is parseable as a Temporal.Absolute and may or may not include an IANA time zone name)
+ * by the corresponding absolute time (e.g., for presenting global log events sequentially).
+ *
+ *
+ * @param {string[]} parseableAbsoluteStrings - a group of ISO Strings
+ * @param {boolean} reverse ascending or descending order
+ * @returns {string[]} Temporal.Absolute instance
```

Comment:
One more thing, this isn't a Temporal.Absolute instance anymore, I'd say something like "the array from parseableAbsoluteStrings, sorted"